motoko/dao/src/Types.mo

```diff
@@ -0,0 +1,65 @@
+import Result "mo:base/Result";
```

Comment:
Does this mean Rust and Motoko will have different candid interface with #Err vs #err?

Reply:
Yes